Source/Common/SchemaEnumJsonConverter{T}.cs

```diff
@@ -41,11 +41,21 @@ public SchemaEnumJsonConverter()
     /// <returns>The enumeration value.</returns>
     public override T Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
     {
+#if NET6_0_OR_GREATER
+        ArgumentNullException.ThrowIfNull(typeToConvert);
+        ArgumentNullException.ThrowIfNull(options);
+#else
         if (typeToConvert is null)
         {
             throw new ArgumentNullException(nameof(typeToConvert));
         }
 
+        if (options is null)
+        {
+            throw new ArgumentNullException(nameof(options));
+        }
```

Comment:
Does it make sense to assert this if we don't actually use `options`?

Reply:
Good catch. Got overzealous here. Removed.